webapp/components/admin_console/compliance_settings.jsx

```diff
@@ -0,0 +1,275 @@
+                                defaultMessage='Enable Compliance:'
+                            />
+                        </label>
+                        <div className='col-sm-8'>
+                            <label className='radio-inline'>
+                                <input
+                                    type='radio'
+                                    name='Enable'
+                                    value='true'
+                                    ref='Enable'
+                                    defaultChecked={this.props.config.ComplianceSettings.Enable}
+                                    onChange={this.handleEnable}
+                                    disabled={!licenseEnabled}
+                                />
+                                <FormattedMessage
+                                    id='admin.compliance.true'
+                                    defaultMessage='true'
+                                />
+                            </label>
+                            <label className='radio-inline'>
+                                <input
+                                    type='radio'
+                                    name='Enable'
+                                    value='false'
+                                    defaultChecked={!this.props.config.ComplianceSettings.Enable}
+                                    onChange={this.handleDisable}
+                                />
+                                <FormattedMessage
+                                    id='admin.compliance.false'
+                                    defaultMessage='false'
+                                />
+                            </label>
+                            <p className='help-text'>
+                                <FormattedMessage
+                                    id='admin.compliance.enableDesc'
+                                    defaultMessage='When true, Mattermost allows compliance reporting'
+                                />
+                            </p>
+                        </div>
+                    </div>
+
+                    <div className='form-group'>
+                        <label
+                            className='control-label col-sm-4'
+                            htmlFor='Directory'
+                        >
+                            <FormattedMessage
+                                id='admin.compliance.directoryTitle'
+                                defaultMessage='Compliance Directory Location:'
+                            />
+                        </label>
+                        <div className='col-sm-8'>
+                            <input
+                                type='text'
+                                className='form-control'
+                                id='Directory'
+                                ref='Directory'
+                                placeholder={formatMessage(holders.directoryExample)}
```

Comment:
Should probably use `Utils.localizeMessage(id, default)` so that we don't need to do the injectIntl